packages/input-selection/src/util.ts

```diff
@@ -46,6 +47,7 @@ export const mintToImplicitTokens = (mintMap: Cardano.TokenMap = new Map()) => {
 };
 
 export const preProcessArgs = (
+  mustSpendUtxo: Set<Cardano.Utxo>,
```

Comment:
How about? :thinking: 
```suggestion
  preSelectedUtxo: Set<Cardano.Utxo>,
```

Reply:
done here https://github.com/input-output-hk/cardano-js-sdk/pull/1299/commits/96084800dbad4c99d1762eee7bfd07bb5f212b45

Reply:
There are some lint errors for object key order and please also replace 'must spend' in test names